Formula/passenger.rb

```diff
@@ -81,5 +102,35 @@ def caveats; <<~EOS
   test do
     ruby_libdir = `#{HOMEBREW_PREFIX}/bin/passenger-config --ruby-libdir`.strip
     assert_equal "#{libexec}/src/ruby_supportlib", ruby_libdir
+
+    (testpath/"nginx.conf").write <<~EOS
+      load_module #{opt_libexec}/buildout/nginx_dynamic/ngx_http_passenger_module.so;
+      worker_processes 4;
+      error_log #{testpath}/error.log;
+      pid #{testpath}/nginx.pid;
+
+      events {
+        worker_connections 1024;
+      }
+
+      http {
+        passenger_root #{opt_libexec}/src/ruby_supportlib/phusion_passenger/locations.ini;
+        passenger_ruby /usr/bin/ruby;
+        client_body_temp_path #{testpath}/client_body_temp;
+        fastcgi_temp_path #{testpath}/fastcgi_temp;
+        proxy_temp_path #{testpath}/proxy_temp;
+        scgi_temp_path #{testpath}/scgi_temp;
+        uwsgi_temp_path #{testpath}/uwsgi_temp;
+
+        server {
+          passenger_enabled on;
+          listen 8080;
+          root #{testpath};
+          access_log #{testpath}/access.log;
+          error_log #{testpath}/error.log;
+        }
+      }
+    EOS
+    system "nginx", "-t", "-c", testpath/"nginx.conf"
```

Comment:
This is failing as it's trying to write to places it cannot: https://jenkins.brew.sh/job/Homebrew%20Core%20Pull%20Requests/37812/version=mojave/testReport/junit/brew-test-bot/mojave/test_passenger/

Reply:
That's weird, as the `proxy_temp_path` and `error_log` paths are both set to sub-directories of the `testpath` in the config passed to nginx.

It also passes on my system, which might mean that the sandbox isn't active when I run `brew test`? Is there a way to turn that on locally so I can try and figure this out?

Reply:
Not sure of anything beyond the test failure I'm afraid.

Reply:
Oh, it's maybe because this should be "#{bin}/nginx` too.

Reply:
(or at least a fully specified path)

Reply:
Actually i just remembered a [passenger issue](https://github.com/phusion/passenger/issues/2075) that may be related I'll look into that and get back to you.

Reply:
Um in the test `#{bin}/nginx` gives `/usr/local/Cellar/passenger/6.0.1/bin/nginx`, which isn't right. Is there a different helper for "brew home bin"?

Reply:
@CamJN Sorry, you're right, I got confused. `#{Formula["nginx"].opt_bin}/nginx` is what you want.

Reply:
Ok things are looking good. I've merged the patch upstream: https://github.com/phusion/passenger/commit/7d4251bb4aec5cef6896f6d1905a30e99d21befe and will release 6.0.2 soon with it included. I'm happy with this, hope it can move forward.